app.py

```diff
@@ -29,9 +35,14 @@ def post():
 
                 prediction = model.predict(model_input)
 
-                model_output = parse_prediction(prediction)
+                prediction_index = parse_prediction(prediction)
 
-                return jsonify(model_output)
+
+                output_json={ "prediction":int(prediction_index) }
+                
+                print(output_json)
+
+                return jsonify(**model_output)
```

Comment:
I think you have removed the model_ouptut parameter but used it here. Am I missing something?